Ukoly/DU_6/Piskvorky.py

```diff
@@ -1,62 +1,63 @@
 from random import randrange
 
-WINNER = None
+winner = None
```

Comment:
Aaaa... prečo to je stále vonku z tej funkcie? :smile: Daj to do tej funkcie `evaluation`, kde to používaš.